grate.unittests/Generic/GenericDatabase.cs

```diff
@@ -29,6 +30,22 @@ public async Task Is_created_if_confed_and_it_does_not_exist()
         databases.Should().Contain(db);
     }
 
+    [Test]
+    public async Task Is_created_with_custom_script_if_confed_and_it_does_not_exist()
+    {
+        var db = "NEWDATABASE";
+
+        var config = GetConfiguration(db, true, "createDatabase.sql");
+
+        TestConfig.WriteContent(config.SqlFilesDirectory, "createDatabase.sql", $"CREATE DATABASE {{DatabaseName}}");
```

Comment:
Could we have this test create a database with a _different_name, or a suffix or something, so that we can assert it was the file that was run, and not just the default behaviour?

Reply:
on the test, I'm creating 2 databases so that you can see the custom script is used.